src/Components/spoiler/spoiler.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import "./spoiler.css";
+
+
+
+export default class Spoiler extends Component{
+    constructor(props){
+        super(props);
+    }
+
+    render(){
+        const {n, question, answer} = this.props;
+        return(
+            <li className="spoiler">
+                <h2 className="question" onClick={(e)=>{
+                    let flag1 = false;
+                    let flag2 = false;
+                    
+                    if (e.currentTarget.classList.contains("focus")){
+                        flag1 = true;
+                    }
+                    let list = e.currentTarget.parentNode.parentNode.querySelectorAll(".focus");
+                    list.forEach((item)=>{
+                        item.classList.remove("focus");
+                    });
+                    const el = document.querySelector("#"+n);
+                    if (el.classList.contains("show")){
+                        flag2 = true;
```

Comment:
still no idea what `flag2` means here. That answer is visible? name it `isAnswerVisible` then